source/common/http/conn_manager_utility.cc

```diff
@@ -47,6 +47,15 @@ Network::Address::InstanceConstSharedPtr ConnectionManagerUtility::mutateRequest
   request_headers.removeProxyConnection();
   request_headers.removeTransferEncoding();
 
+  // When this flag is true, we represent IPv4 addresses as IPv4 mapped IPv6 if
+  //   - use_remote_address is true &&
+  //   - the address was not derived from the existing XFF header &&
+  //   - the address is not a loopback address
+  const bool represent_ipv4_as_ipv6 = runtime.snapshot().featureEnabled(
+      "http_connection_manager.represent_ipv4_remote_address_as_ipv4_mapped_ipv6",
```

Comment:
please make sure to doc this in your follow up doc/release note PR.

Reply:
it's already in the docs but i will mention in release notes

https://github.com/envoyproxy/data-plane-api/pull/467/files#diff-941216ab10522bf09c0cd2fe6a752a4bR8